.github/workflows/e2e-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Reusable e2e Workflow
```

Comment:
Let's rename this file `e2e-tests.yml` to mimic the unit tests file naming.

Reply:
Done!